data/tobySolutions.json

```diff
@@ -134,6 +134,14 @@
       "color": "blue",
       "description": "I attended the Microsoft Learnathon organized by the ADC ecosystem and hosted at the Microsoft Nigeria's branch",
       "url": "https://twitter.com/toby_solutions/status/1623026397055680512"
+    },
+    {
+      "title": "Became Web team volunteer PAADC Cohort 7.0",
+      "date": "2023",
+      "icon": "CgCommunity",
```

Comment:
I don't think that is a valid icon, it won't block the PR but will default to another icon

Reply:
You can search for available icons here http://linkfree.io/icons

Reply:
Thank you Eddie!